datasource/index.js

```diff
@@ -198,7 +198,14 @@ module.exports = yeoman.generators.Base.extend({
 
   installConnector: function() {
     var connector = this.availableConnectors[this.connector];
-    var pkg = connector.package;
+    var pkg;
+    if (!connector) {
+      pkg = {
+        name: this.connector
+      };
+    } else {
+      pkg = connector.package;
+    }
     if (!pkg) return;
```

Comment:
While we are changing this code, can we also add support for installing packages for connectors that do not provide `package` entry? I mean when no package name is provided, we should `npm install` the connector name. This should apply both for known connectors and unknown connectors too.